### PR TITLE
docs: add TOC to setup.md

### PR DESCRIPTION
**Pull request by dougsland**
- Add Table of Content to setup.md
- Minor adjustments

Signed-off-by: Douglas Schilling Landgraf <dougsland@redhat.com>

#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:

Add to setup.md Table of Content

### CODE REVIEW

File: tutorials/setup.md [hunk 1]
```diff
@@ -1,3 +1,24 @@
+- [Build and install CRI-O from source](#build-and-install-cri-o-from-source)
```
Comment (TomSweeneyRedHat) [inline tutorials/setup.md]: I know it wasn't there before, but could we add the CRI-O logo to the top of this page?
`CRI-O logo`

Comment (TomSweeneyRedHat) [reply tutorials/setup.md]: oops, the above probably needs to be `../logo/crio-logo.svg`

**Comment by TomSweeneyRedHat** [comment]
LGTM
Nice change, if you decide to add the logo it could be done later as this has danced through the test process already.

File: tutorials/setup.md (hunk 1 above)
Comment (dougsland) [reply tutorials/setup.md]: Sure Tom. Looks good now. Just updated the PR.

**Comment by TomSweeneyRedHat** [comment]
LGTM once the DCO stuff is happy.

**Comment by dougsland** [comment]
git rebase -i && squash doesn't work well. Re-sending.

**Pull request closed without merging**